src/TrixiShallowWater.jl

```diff
@@ -1,10 +1,15 @@
 module TrixiShallowWater
-
+# Decide between using Trixi: Trixi, import Trixi or using Trixi?
```

Comment:
We should update this comment and (maybe?) add a link to the resolved conversation below for future reference.

Reply:
Yes, that is a good idea to reference this. I have adjusted the comment and added a reference to the github conversation in d256bc72604dcf18bb39384320c90b135fe551f1